### PR TITLE
DOCS-5103 Add Quay.io access

### PR DESCRIPTION
**Pull request by anxumalo**
* Added 'docker login quay.io' for Docker Compose steps
* Added new section for Helm deployment - 'Creating a Docker registry pull secret'
* Updated Helm steps to include 'quay-registry-secret'

### CODE REVIEW

File: docs/helm-deployment-aws_cloud.md [hunk 1]
```diff
@@ -328,6 +328,49 @@ export EFS_SERVER=$EFS_FS_ID.efs.$AWS_DEFAULT_REGION.amazonaws.com
...
+kubectl create -f secrets.yaml --namespace $DESIREDNAMESPACE
+```
+* You should see the following output:
+```bash
+secret "quay-registry-secret" created
+```
+
+* When installing the ACS Helm chart, add a variable:
+```bash
+--set registryPullSecrets=quay-registry-secret
```
Comment (montgolfiere) [inline docs/helm-deployment-aws_cloud.md]: Also add at line 447 - example command for Deploying Alfresco Content Services below

Comment (DenisGabriela) [reply docs/helm-deployment-aws_cloud.md]: It should be added on each example command (there are 4 examples in this doc)

File: docs/helm-deployment-aws_cloud.md [hunk 2]
```diff
@@ -328,6 +328,56 @@ export EFS_SERVER=$EFS_FS_ID.efs.$AWS_DEFAULT_REGION.amazonaws.com
...
+# Linux
+cat ~/.docker/config.json | base64
+```
+
+```bash
+# Windows
+base64 -w 0 ~/.docker/config.json
+```
+
+You can also generate your pull secret and download it from your Quay.io account. Select **Account Settings** > **Generate Encrypted Password** > **Kubernetes Secret**.
```
Comment (anxumalo) [inline docs/helm-deployment-aws_cloud.md]: @killerboot I'll remove this line, in case the options are not accessible from robot/service accounts.

**Comment by montgolfiere** [comment]
@killerboot I assumed the docker login quay.io username/password (as used for docker-compose) also gives access to quay.io website. However, someone would need to double-check from customer/partner issued login details.

**Comment by anxumalo** [comment]
@montgolfiere Just confirmed with Support - you can’t even log in to Quay using the robot account credentials - it only works from the command-line.

**Comment by montgolfiere** [comment]
@anxumalo Good to know - thanks for checking.

**Pull request merged**